### PR TITLE
Feature/toolboxes

### PR DESCRIPTION
**Pull request by vincentchabannes**
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/feelpp/feelpp/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes?
- [ ] Have you successfully run the Feel++ testsuite with your changes locally?
- [ ] Have you written Doxygen comments in your contribution ?

-----

### CODE REVIEW

File: toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp [hunk 1]
```diff
@@ -0,0 +1,843 @@
...
+                }
+            }
+        ret_type
+        evaliq( uint16_type i, uint16_type q ) const override
+            {
+                if constexpr ( this_type::is_applied_as_eval )
+                    return this->evalq( q );
+                else
+                {
+                    CHECK( false) << "not allow";
```
Comment (prudhomm) [inline toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp:152]: throw an exception here

File: toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp [hunk 2]
```diff
@@ -0,0 +1,843 @@
...
+
+
+        value_type
+        evaliq( uint16_type i, uint16_type c1, uint16_type c2, uint16_type q ) const override
+            {
+                if constexpr ( this_type::is_applied_as_eval )
+                    return this->evalq( c1,c2,q );
+                else
+                {
+                    CHECK( false) << "not allow";
```
Comment (prudhomm) [inline toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp:141]: exception

File: toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp (hunk 1 above)
Comment (prudhomm) [reply toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp:152]: or better message ?

**Review by prudhomm — APPROVED** [review]

File: toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp (hunk 1 above)
Comment (vincentchabannes) [reply toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp:152]: this is in an expression tensor, so no exception here

File: toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp (hunk 2 above)
Comment (vincentchabannes) [reply toolboxes/feel/feelmodels/modelvf/fluidmecconvectiveterm.hpp:141]: idem

**Pull request merged**